nodl_python/test/test_pep257.py

```diff
@@ -18,5 +18,22 @@
 @pytest.mark.linter
 @pytest.mark.pep257
 def test_pep257():
-    rc = main(argv=['.'])
+    rc = main(
+        argv=[
+            '.',
+            'test',
+            '--ignore',
+            'D100',
+            'D101',
+            'D102',
+            'D103',
+            'D104',
+            'D105',
+            'D106',
+            'D107',
+            'D203',
+            'D213',
+            'D404',
```

Comment:
What are all these, and why are they only now required?

Reply:
There's no way to add entries onto the default ignore list. We need to provide D213 or else some files complain about docstrings not starting with a line break, so the entire default ignores needed to be provided as well.

Reply:
I'm confused. We have numerous docstrings that don't start with a line break. Why is this only problematic now?

Reply:
I don't know, it's only a problem in certain files as well, and sometimes only on certain functions in those files.

Reply:
Some research.

`generate_pep257_report()` creates the following list of files to check
```
['./setup.py', './test/test_pep257.py', './test/test_mypy.py', './test/test_flake8.py', './test/nodl/test_index.py', './test/nodl/test_types.py', './test/nodl/test_utils.py', './nodl/types.py', './nodl/errors.py', './nodl/__init__.py', './nodl/_index.py', './nodl/_util.py']
```

With the following error codes to check against
```
['D200', 'D201', 'D202', 'D204', 'D205', 'D206', 'D207', 'D208', 'D209', 'D210', 'D211', 'D213', 'D214', 'D215', 'D300', 'D301', 'D302', 'D400', 'D401', 'D402', 'D403', 'D405', 'D406', 'D407', 'D408', 'D409', 'D410', 'D411', 'D412', 'D413', 'D414', 'D415', 'D416', 'D417']
```
(identical for each file).

Reply:
Opened https://github.com/ament/ament_lint/issues/240 , will propose a PR to fix this

Reply:
ament/ament_lint#241 fixes this